docs/configuration.rst

```diff
@@ -67,9 +67,11 @@ Cluster
 +------------------------------------------+------------------------------------------------------------------------------------------------+-------------------+-------------------------------------------------+-----------------+
 | Option                                   | Description                                                                                    | Type              | Default                                         | Client / Server |
 +==========================================+================================================================================================+===================+=================================================+=================+
-| rubix.hadoop.clustermanager.class        | The ``ClusterManager`` class to use for fetching node-related information for Hadoop clusters. | string            | com.qubole.rubix.hadoop2. Hadoop2ClusterManager | S               |
+| rubix.cluster.node.refresh.time          | The frequency at which the cluster node membership will be checked                             | integer (s)       | 300 sec                                         | C & S           |
 +------------------------------------------+------------------------------------------------------------------------------------------------+-------------------+-------------------------------------------------+-----------------+
-| rubix.presto.clustermanager.class        | The ``ClusterManager`` class to use for fetching node-related information for Presto clusters. | string            | com.qubole.rubix.presto. PrestoClusterManager   | S               |
+| rubix.clustermanager.hadoop.class        | The ``ClusterManager`` class to use for fetching node-related information for Hadoop clusters. | string            | com.qubole.rubix.hadoop2. Hadoop2ClusterManager | C & S           |
```

Comment:
For consistency with other cluster-related options, we could prefix these cluster manager options with `rubix.cluster.manager...` instead